src/Components/UserInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Post } from '../types/Post';
+import { CommentList } from './CommentList';
+
+export type Props = {
+  post: Post;
+};
+
+export const UserInfo: React.FC<Props> = ({ post }) => (
+  <div className="post__user user">
+    <div className="user__name">
+      <strong>Name: </strong>
+      {post.userName[0].name}
+    </div>
+    <div className="user__email">
+      <strong>E-mail: </strong>
+      {post.userName[0].email}
+    </div>
+    <CommentList {...post} />
```

Comment:
```suggestion
    <CommentList {...post} />
```
Компонент CommentList логічніше рендерити у компоненті PostInfo(поруч із UserInfo)
Коменти відносяться до поста, а не до юзера